modules/upgrade-central.adoc

```diff
@@ -10,10 +10,6 @@ You can update Central to the latest version by downloading and deploying the up
 .Prerequisites
 
 * If you deploy images from a private image registry, first push the new image into your private registry, and then replace your image registry for the commands in this section.
-* If you used Red Hat UBI-based images when you installed {product-title}, replace the image names for the commands in this section with the following UBI-based image names:
-** For Central, Sensor, and Compliance, use `\registry.redhat.io/rh-acs/main-rhel`
-** For Scanner, use `\registry.redhat.io/rh-acs/scanner-rhel` and `\registry.redhat.io/rh-acs/scanner-db-rhel`
-** For Collector, use `\registry.redhat.io/rh-acs/collector-rhel`
 
 .Procedure
```

Comment:
(outside the scope of this change, so feel free to ignore of file away for later)
This seems like many steps buried in one "step" - might be a nice enhancement to separate these out and say what each step is actually doing. I recognize that this format is not new to this PR, and that you may not have bandwidth to do so, but just putting it out there for your consideration :)